Chat_App/Core/Custom UI/TextView/TextView.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  TextView.swift
+//  Chat_App
+//
+//  Created by Ana Dzamelashvili on 5/11/23.
+//
+
+import Foundation
+import UIKit
+
+protocol ButtonActionDelegate: AnyObject {
+    func buttonTapped(sender: UIButton)
+}
+protocol TextViewDelegate: AnyObject {
+    func changeTextColor(color: UIColor)
+}
+
+class TextView: UIView {
+    
+    private let lineHeight: CGFloat = Constants.TextView.font.lineHeight
+    private lazy var sendButton = UIButton()
+    private lazy var containerView = UIView()
+    lazy var textView = UITextView()
+    weak var delegate: ButtonActionDelegate?
+    weak var textViewDelegate: TextViewDelegate?
+    
+    
+    private let placeholder: UILabel = {
+        let placeholder = UILabel()
+        placeholder.text = "დაწერე შეტყობინება..."
+        placeholder.font = Constants.TextView.font
+        placeholder.sizeToFit()
+        
+        placeholder.textColor = Constants.Colors.placeholderColor
+        
+        placeholder.translatesAutoresizingMaskIntoConstraints = false
+        return placeholder
+    }()
+    
+    override init(frame:CGRect) {
+        super.init(frame: .zero)
+        setUpLayout()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func getSendButton() -> UIButton {
+           return sendButton
```

Comment:
omit return

Reply:
or alternatively you can make `sendButton`'s getter public and setter private (`public private(set)`) and get rid of that method